README.adoc

```diff
@@ -69,6 +69,20 @@ A workaround was implemented to provide JGit but use Apache HTTP client for auth
 The issue in JGit has now been resolved and delivered in git client plugin releases.
 JGit with Apache HTTP Client continues to delivered to assure compatibility.
 
+[#installing-portablegit-automatically]
+=== Installing PortableGit Automatically
+The git client plugin can install link:https://git-scm.com/download/win[Git for Windows Portable] automatically from a zip file.
+
+* Download and install link:https://github.com/git-for-windows/git/releases/[Git for Windows Portable] from its 7z.exe file
+* Create a zip file of the installation as `PortableGit-a.bb.c.zip`
```

Comment:
Proposed additional comment to assure the zip file contains the expected layout.

```suggestion
* Create a zip file of the installation as `PortableGit-a.bb.c.zip`.
  The root of the zip file should contain a directory named `PortableGit-a.bb.c` so that the entire git installation is inside a single directory in the zip file
```